interfaces/reader_test.go

```diff
@@ -5,7 +5,9 @@ package interfaces
 
 import (
 	"bytes"
-	oldcsv "encoding/csv"
+
+	oldcsv "github.com/eltorocorp/go-csv"
```

Comment:
You shouldn't need to alias this as `oldcsv` Just `github.com/eltorocorp/go-csv' will be fine.